src/app/components/feed/FeedHeader.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { createFragmentContainer, graphql } from 'react-relay/compat';
+import { browserHistory } from 'react-router';
+import IconButton from '@material-ui/core/IconButton';
+import SettingsOutlinedIcon from '@material-ui/icons/SettingsOutlined';
```

Comment:
Please change this to the settings.svg we have in the icons folder

Reply:
Done here: https://github.com/meedan/check-web/commit/d45bfc0253b418cd0293b1bd1267300dde646709